src/platform/export/fileConverter.ts

```diff
@@ -111,7 +111,7 @@ export class FileConverter implements IFileConverter {
             // As far as this method is concerned the export was successful, whether the user opens the file or not
             // should not have any bearing on the completion (resolving) this method.
             // Hence don't await.
```

Comment:
I should update this comment, but I don't understand why the tests don't fail before.

Reply:
Cause Don had added a 'suite.only' to a different test. So only his test was running. The export test wasn't running at all.

See this PR where Don fixes that:
https://github.com/microsoft/vscode-jupyter/pull/10136#pullrequestreview-985488189

Reply:
You probably need his change to verify this is working.

Reply:
I don't think we need this, I hvae a fix, didn't realize my PR was meged, I'll submit a PR with the fxies.

Reply:
There are a few other bugs in export (basically it never uses nbconvert today)

Reply:
Here's the PR https://github.com/microsoft/vscode-jupyter/pull/10140

Reply:
> Cause Don had added a 'suite.only' to a different test. So only his test was running. The ex

Whats interesting is i added that skip only yesterday, but these tests have been around before then. weird.